modules/retail/pages/retail-components.adoc

```diff
@@ -104,15 +107,33 @@ You can use formulas to apply configuration patterns to your hardware groups.
 
 You can use default settings for formulas, or edit them to make them more specific to your environment.
 
-For more information about formulas, see xref:retail:retail-formulas.adoc[].
+For more information about formulas, see xref:retail:retail-formulas-intro.adoc[].
 
 
 
-[[retail.sect.components.fitting.system-groups]]
-=== System groups
+=== Saltboot
 
-{smr} uses system groups to organize the various devices in your environment.
+Saltboot is a collection of tools and processes that are used to bootstrap, deploy and validate {smr} terminals.
 
-Each branch requires a system group, containing a single branch server, and the PoS terminals associated with that server.
-Each system group is identified with a Branch ID.
-The Branch ID is used in formulas and scripts to automatically update the entire group.
+Saltboot consists of:
+
+* Initialization service:
++
+The Saltboot ``initrd`` service is created during image building and is required for bootstrapping terminals.
```

Comment:
I wouldn't call `initrd` a service. It is a bunch of scripts and tools which are run right after kernel is loaded and are meant to prepare the machine to some state before system image takes over. (see https://en.wikipedia.org/wiki/Initial_ramdisk ).
I would really just call it `initrd`, in this case `saltboot initrd`.

Reply:
huh, i always thought the d was for daemon, didn't realise it was RAM disk! That makes a lot more sense now :wink: